client/dashboard/domains/domain-connection-setup/domain-connection-setup.tsx

```diff
@@ -59,16 +61,35 @@ export default function DomainConnectionSetup( {
 		false,
 	] );
 
-	const suggestedModeSteps = [
+	const isReseller = !! domainConnectionSetupInfo.reseller;
+	const registrar = domainConnectionSetupInfo.reseller || domainConnectionSetupInfo.registrar;
+	const registrar_url = domainConnectionSetupInfo.registrar_url;
+
+	const commonSteps = [
 		{
-			title: __( '1. Login to your domain name provider' ),
+			title: registrar
+				? sprintf(
+						// translators: %s is the registrar name
+						__( '1. Login to %s' ),
+						registrar
+				  )
+				: __( '1. Login to your domain name provider' ),
 			label: __( 'I have opened the DNS settings' ),
 			content: (
 				<Text>
-					{ sprintf(
+					{ createInterpolateElement(
 						// translators: %s is the domain name
```

Comment:
```suggestion
						// translators: <registrar/> is the domain's registrar
```
I think we should update the translator comment too 🙂